src/cli-util.js

```diff
@@ -542,29 +543,36 @@ function groupBy(array, getKey) {
   }, Object.create(null));
 }
 
-function normalizeArgv(rawArgv, options) {
+/** @param {'api' | 'cli'} type */
+function normalizeConfig(type, rawConfigs, options) {
```

Comment:
Oops, sorry, the "plural confusion" applies to `rawConfigs` as well – I think `rawConfig` would be better ... sorry for all the nitpicks :(